job_export.py

```diff
@@ -116,4 +129,40 @@
         fname,
     )
 
-    print("[out] {}".format(s3file))
+
+    if access_format == "guid":
+        # calculate md5 sum
+        md5_sum = hashlib.md5()
+        chunk_size = 8192
+        with open(fname, "rb") as f:
+            while True:
+                data = f.read(chunk_size)
+                if not data:
+                    break
+                md5_sum.update(data)
+
+        md5_digest = md5_sum.hexdigest()
+
+        hostname = os.environ["GEN3_HOSTNAME"]
+        COMMONS = "https://" + hostname + "/"
+
+        # try sending to indexd
+        with open("/indexd-creds.json") as indexd_creds_file:
+            indexd_creds = json.load(indexd_creds_file)
+
+        s3_url = "s3://" + pelican_creds["manifest_bucket_name"] + "/" + avro_filename
+
+        indexd_record = indexd_submit(
+            COMMONS,
+            indexd_creds["user_db"]["gdcapi"],
+            avro_filename,
+            os.stat(fname).st_size,
+            [s3_url],
+            {"md5": str(md5_digest)}
+        )    
+
+        # send s3 link and information to indexd to create guid and send it back
+
+        print("[out] {}".format("index/index/"+indexd_record["did"]))
```

Comment:
can u return the GUID without the url prefix actually? windmill has all the URL paths in a config file already, so it can produce a link there

Reply:
I have changed this to output the guid directly